cql/pom.xml

```
@@ -27,6 +27,12 @@
       <version>0.0.6-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.stargate.health</groupId>
+      <artifactId>health-checker</artifactId>
+      <version>0.0.6-SNAPSHOT</version>
```

```suggestion
      <version>0.0.7-SNAPSHOT</version>
```

these all should be bumped to 0.0.7 now.